dataminer/Functions/DOM/DOM_DevOps/DOM_Editor/DOM_editor_attachments_networkshare.md

```diff
@@ -0,0 +1,42 @@
+---
+uid: DOM_editor_attachments_networkshare
```

Comment:
Maybe a reference could be added from the DomInstanceNetworkAttachmentSettings page. This already mentions the DOM editor tool, so we could point to this page instead.